cmake/configure/configure_1_mpi.cmake

```diff
@@ -28,7 +28,23 @@ MACRO(FEATURE_MPI_FIND_EXTERNAL var)
         "Could not find a sufficient MPI version: "
         "Your MPI implementation must define MPI_SEEK_SET.")
       SET(MPI_ADDITIONAL_ERROR_STRING
-        "Your MPI implementation must define MPI_SEEK_SET.")
+        "Your MPI implementation must define MPI_SEEK_SET.\n")
+      SET(${var} FALSE)
+    ENDIF()
+
+    CHECK_COMPILER_SETUP("${MPI_CXX_FLAGS}" "${MPI_LINKER_FLAGS}"
+      MPI_WORKING_COMPILER
+      ${MPI_LIBRARIES}
+      )
```

Comment:
Shouldn't we also include `${DEAL_II_CXX_FLAGS}` and `${DEAL_II_LINKER_FLAGS}` for the first and second argument, respectively? Otherwise, it won't include the offending `-fuse-ld=gold` part here.